rest_framework_ccbv/renderers.py

```diff
@@ -50,16 +53,16 @@ def get_context(self):
             version
             for version in context['other_versions']
             if version in available_versions]
-        context['name'] = self.klass
+        context['name'] = self.klass_name
         context['ancestors'] = self.inspector.get_klass_mro()
         context['direct_ancestors'] = self.inspector.get_direct_ancestors()
         context['attributes'] = self.inspector.get_attributes()
         context['methods'] = self.inspector.get_methods()
 
-        context['this_klass'] = next(
-            filter(lambda x: x.__name__ == self.klass, self.grouped_klasses[self.module])
-        )
-        context['this_module'] = self.module
+        context['this_module'] = self.module_name
+
+        module = importlib.import_module(self.module_name)
```

Comment:
Since the `Field` detail page will still exist, we must provide a way for the script to import its data without having to rely on `grouped_klasses`.